repose-aggregator/components/filters/cors-filter/src/main/scala/org/openrepose/filters/cors/CorsFilter.scala

```diff
@@ -178,23 +204,71 @@ class CorsFilter @Inject()(configurationService: ConfigurationService)
         Nil
     })
   }
+
+  def getHostUri(request: HttpServletRequestWrapper): URI = {
+    val maybeForwardedHost = request.getSplittableHeaderScala(CommonHttpHeader.X_FORWARDED_HOST).headOption
```

Comment:
Ah, I see. So the user must be using a browser that conforms with the CORS specification to get any benefit from it. However, using such a browser is what makes CORS "secure" as it were.